sdk/communication/azure-communication-phonenumbers/README.md

```diff
@@ -103,20 +103,14 @@ The Phone Number Client supports a variety of long running operations that allow
 ### Search for Available Phone Numbers
 Search for available phone numbers by providing the area code, assignment type, phone number capabilities, phone number type, and quantity. The result of the search can then be used to purchase the numbers. Note that for the toll-free phone number type, providing the area code is optional.
 
-<!-- embedme ./src/samples/java/com/azure/communication/phonenumbers/ReadmeSamples.java#L98-L114-->
+<!-- embedme ./src/samples/java/com/azure/communication/phonenumbers/ReadmeSamples.java#L97-L107-->
 ```java
-PhoneNumberSearchRequest searchRequest = new PhoneNumberSearchRequest();
-searchRequest
-    .setAreaCode("800") // Area code is optional for toll free numbers
-    .setAssignmentType(PhoneNumberAssignmentType.PERSON)
-    .setCapabilities(new PhoneNumberCapabilities()
-        .setCalling(PhoneNumberCapabilityValue.INBOUND)
-        .setSms(PhoneNumberCapabilityValue.INBOUND_OUTBOUND))
-    .setPhoneNumberType(PhoneNumberType.GEOGRAPHIC)
-    .setQuantity(1); // Quantity is optional, default is 1
+PhoneNumberCapabilities capabilities = new PhoneNumberCapabilities()
+    .setCalling(PhoneNumberCapabilityType.INBOUND)
+    .setSms(PhoneNumberCapabilityType.INBOUND_OUTBOUND);
 
 PhoneNumberSearchResult searchResult = phoneNumberClient
-    .beginSearchAvailablePhoneNumbers("US", searchRequest, Context.NONE)
+    .beginSearchAvailablePhoneNumbers("US", PhoneNumberType.TOLL_FREE, PhoneNumberAssignmentType.PERSON, capabilities, "800", 1, Context.NONE)
```

Comment:
What I suggested to do in C# is to have all the mandatory parameters here and move the optional ones to an options type, PhoneNumberSearchOptions Azure/azure-sdk-for-net#18876. I think we should similar a similar thing here as well.

Reply:
Thanks @RezaJooyandeh, I've added PhoneNumberSearchOptions.